Initial commit

AkRampersad opened the pull request:
This will commit and push the parquet-converter module to parquet-converter branch of my oakvar-module repo. This is also the first pull request from Akash Rampersad.

AkRampersad committed:
Initial commit

This will commit and push the parquet-converter module to parquet-converter branch of my oakvar-module repo

rkimoakbioinformatics commented:
**parquet-converter.yml**
- Please add the following.
```
no_data: true
format_name: parquet
requires_oakvar: "2.8.34"
```
- `developer` field: Use one time only, such as something below.
```
developer:
  module:
    name: "Akash Rampersad, Ryangguk Kim"
    ...
```

**parquet-converter.py**
- With ov 2.8.34, `convert_file` will receive input_path variable, which is the str of the path to the input file. Please use `convert_file`'s signature as shown below, and use `input_path` in duckdb connection.
```
def convert_file(
>>        self, file, input_path: str="", *__args__, exc_handler=None, **__kwargs__
      ) -> Iterator[Tuple[int, List[dict]]]:
```

**Test**
- Use https://b1q6.c19.e2-1.dev/oakvar-modules/temp/sample.2.parquet for test. It is smaller (about 10 variants) and thus better for quick development and test.
- Make sure that `ov run sample.2.parquet` runs without an error. Before submitting a pull request, the code should be tested and run without an error.

AkRampersad committed:
Input_path and yml update

changed the convert_file method to include input_path
changed duckdb query to utilize input_path
fixed .yml formatting

rkimoakbioinformatics commented:
- Add the following to the parquet-converter.py inside the class definition:
```Python
    def __init__(self, *args, **kwargs):                                                             
          super().__init__(*args, **kwargs) 
          self.format_name = "parquet"
```
- Check if input_path is used throughout, not a file object in the code.

AkRampersad committed:
Update parquet-converter.py

removed asterik from query command
